### PR TITLE
Improve cover documentations

### PR DESCRIPTION
**Pull request by breeswish**
Many documentations are missing. This PR improves the documentation quality for the cover page (i.e. http://docs.rs/prometheus ).

### CODE REVIEW

**Comment by breeswish** [comment]
@overvenus @siddontang PTAL

**Comment by overvenus** [comment]
LGTM

File: src/macros.rs [hunk 1]
```diff
@@ -165,7 +166,7 @@ macro_rules! __register_counter {
     };
 }
 
-/// Create a `Counter` and register to default registry.
+/// Creates a `Counter` and register to default registry.
```
Comment (siddontang) [inline src/macros.rs]: registers

**Comment by siddontang** [comment]
LGTM

**Pull request merged**